app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,48 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LittleShopV2</title>
+    <title>Art Thieves</title>
     <%= csrf_meta_tags %>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
This is solid, we'll need to wrap this in an if statement for pages we don't want the navbar. Making a note in the project page.

Reply:
Done.